libraries/botframework-connector/src/teams/operations/teams.ts

```diff
@@ -9,11 +9,13 @@ import * as Models from '../models';
 import * as Mappers from '../models/teamsMappers';
 import * as Parameters from '../models/parameters';
 import { TeamsConnectorClientContext } from '../';
-import { ConversationList, TeamDetails, TeamsMeetingInfo, TeamsMeetingParticipant, MeetingNotificationResponse, MeetingNotification } from 'botframework-schema';
+import { Activity, ConversationList, TeamDetails, TeamsMeetingInfo, TeamsMeetingParticipant, MeetingNotificationResponse, MeetingNotification, TeamsMember, BatchOperationResponse, GetOperationStateResponse, GetFailedEntriesResponse } from 'botframework-schema';
+import { retry } from '../';
 
 /** Class representing a Teams. */
 export class Teams {
     private readonly client: TeamsConnectorClientContext;
+    private readonly retryCount = 7;
```

Comment:
In dotnet is using 7 attempts ? or lower?

Reply:
![image](https://github.com/southworks/botbuilder-js/assets/122501764/7755a402-b2c5-4fdb-b0fb-4e3ee7e8f662)